src/views/open-api-docs-view/utils/get-operation-props.ts

```diff
@@ -36,22 +36,38 @@ export async function getOperationProps(
 				continue
 			}
 
+			// Create a slug for this operation
+			const operationSlug = slugify(paramCase(operation.operationId), {
+				lower: true,
+			})
+
 			/**
 			 * Parse request and response details for this operation
 			 */
 			const parameters = 'parameters' in operation ? operation.parameters : []
-			const requestData = await getRequestData(
-				parameters,
-				operation.requestBody
-			)
-			const responseData = await getResponseData(operation.responses)
+			const requestDataSlug = `${operationSlug}_request`
+			const requestData = {
+				heading: { text: 'Request', slug: requestDataSlug },
+				noGroupsMessage: 'No request data.',
+				groups: await getRequestData(
+					parameters,
+					operation.requestBody,
+					requestDataSlug
```

Comment:
A big part of the work in this PR was passing through `parentSlug` arguments to allow everything in the hierarchy to have a unique slug. Each slug is then structured like `<parentSlug><slug>`.

Above the level of nested properties, we separate `<parentSlug>` and `<slug>` with an `_` character, as `operationId` values are `slugified` and will never contain `_`.

At the property level, we separate `<parentSlug>` and `<slug>` with a `.` character, as property keys are never expected to contain `.` characters that cause collisions.

To be more concrete, we have guaranteed uniqueness at the `operationId` level, as well as at each level where we need slugs, so we progressively build off of that:
    - Operation headings, such as `ListApps`, use the `operationId` (as they did previously)
    - Property sections use the above as the `parentSlug`
        - Property sections are guaranteed unique, we have only two - `request` and `response`
    - Property groups use the above as the `parentSlug`
        - Property groups are guaranteed unique
        - Under `request`, we have `path`, `query`, and `body`
        - Under `response`, we have `default`, plus response codes like `200`, `404`, `500`, etc
    - Top-level properties use the above as the `parentSlug`
        - Properties are unique within an object
    - Nested properties at arbitrary depths use their parent property
        - Again, properties are unique within an object

All of the work done in `src/views/open-api-docs-view/utils` files is to achieve the above implementation of slug uniqueness 👍

Reply:
This is really clever, I was wondering how you were going to guarantee uniqueness. Transferring some of this context to a code comment would be helpful!